app/routes/invitations.rb

```diff
@@ -3,11 +3,11 @@ module Routes
     class Invitations < Core
       post '/teams/:slug/invitations' do |slug|
         please_login
+        valid_invitee = params[:usernames].delete(current_user.username)
         only_for_team_managers(slug, "You are not allowed to add team members.") do |team|
-          team.invite_with_usernames(params[:usernames], current_user)
+          team.invite_with_usernames(valid_invitee, current_user)
```

Comment:
Sure. I understand... but... I usually think of routes/controllers (like `teams.rb`) to only be for handling frontend/session/security related things. I wouldn't really expect the route to handle team invitation logic.

I'm saying I think that could be done better [here](https://github.com/exercism/exercism.io/blob/3de39152058110ace5b1ea98f3f9915d13480294/lib/exercism/team.rb#L73) where we actually create the invitations, have both the invitees and inviter and have already filtered out members that are already in the team.

Just my two cents.